lib/jekyll/commands/build.rb

```diff
@@ -57,7 +57,7 @@ def build(site, options)
           Jekyll.logger.info "Incremental build:", (full_build ? "disabled" : "enabled")
           Jekyll.logger.info "Generating..."
           process_site(site)
-          Jekyll.logger.info "", "done in #{Time.now - t} seconds."
+          Jekyll.logger.info "", "done in #{(Time.now - t).round 2} seconds."
```

Comment:
in the case of your possible world. then we need to use ceil(), so it will be 0.01  :-D

i thought no parenthesis was more aesthetic.
anyway.. fixed